models/kube/frontend/deployment/deployment_strategy.k

```diff
@@ -0,0 +1,25 @@
+"""
+This is the deployment_strategy module in k8s.api.apps.v1 package.
+This file was generated by the KCL auto-gen tool. DO NOT EDIT.
+Editing this file might prove futile when you re-run the KCL auto-gen generate command.
+"""
+
+
+schema DeploymentStrategy:
+    """
+    DeploymentStrategy describes how to replace existing pods with new ones.
+
+    Attributes
+    ----------
+    rollingUpdate : RollingUpdateDeployment, default is Undefined, optional
+        Rolling update config params. Present only if DeploymentStrategyType = RollingUpdate.
+    $type : str, default is Undefined, optional
```

Comment:
default is Unfined or RollingUpdate?

Reply:
The doc is generated. `Unfined` is the default value of `type`, and `RollingUpdate` is the default status of the k8s cluster.